src/components/DistanceRequest.js

```diff
@@ -170,6 +180,7 @@ function DistanceRequest({iou, iouType, report, transaction, mapboxAccessToken})
                     onContentSizeChange={(width, height) => setScrollContentHeight(height)}
```

Comment:
I totally agree about remove useEffect.
Good idea

But what about the height
I think it's better to compare the number of products, because for example when opening the stop details
The height will reset and scroll will occur, which we do not need